static/dbstructure.config.php

```diff
@@ -1019,15 +1019,18 @@
 	"parsed_url" => [
 		"comment" => "cache for 'parse_url' queries",
 		"fields" => [
-			"url" => ["type" => "varbinary(255)", "not null" => "1", "primary" => "1", "comment" => "page url"],
+			"url_hash" => ["type" => "char(64)", "not null" => "1", "primary" => "1", "comment" => "page url hash"],
```

Comment:
I would suggest using `BINARY` instead of `CHAR`. We don't need collation here. This would also reduce the key size and increase the fetch speed.

Reply:
It is indeed a hexadecimal value, but does the comparison with strings still stands with a binary column?

Reply:
The `item-uri` table uses a `varbinary` for the `uri` and we can fetch and store it like a string.

The difference of the `binary` types vs. the `char` types is that a `binary(64)` stores 64 bytes, while a `char(64)` stores 64 characters - but since it has to support multi byte characters, it stores 4 bytes per character, so a `char(64)` stores 256 bytes.

In `char` the string `ABCD` is equal to `abcd` and a `ä` can be equal to an `a`, depending on your collation.

With a hash you don't need 4 byte unicode storage, you don't need support for umlauts, ...

Reply:
Ok, I just needed an assurance that I don't need to change my conditions.